src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-breadcrumb {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+
+  > .iui-button {
+    margin-right: $iui-s;
+  }
+  
+  ol {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    li {
+      white-space: nowrap;
+
+      a:not([aria-current]) {
+        @include iui-anchor;
+      }
+
+      [aria-current] {
+        text-decoration: none;
+        @include iui-focus;
+        @include themed {
+          color: t(iui-text-color);
+        }
+      }
+
+      &:not(:first-child) {
+        margin-left: $iui-m;
```

Comment:
Maybe apply it to the splitter.

Reply:
Oh yeah, that makes way more sense!  Updated.